main.py

```diff
@@ -239,224 +45,42 @@ def get_query(self) -> str:
                           }
                        }
                     }
+                  UserQuery{
+                    all{
+                        nodes{
+                            email
+                            userName
+                            roles
+                        }
+                    }
+                  }
 
                 }
             """
 
-    def handle_json_response(self, json_response):
-        pass
-
-
-class RouteDeleter(Query):
-    def __init__(self, route_id: int, endpoint, login_cookie):
-        super().__init__(endpoint, login_cookie)
-        self.route_id = route_id
-
-    def get_query(self) -> str:
-        return Template("""
-            mutation DeleteRoute{
-              RouteMutation{
-                deleteRoute(routeId: $routeId){
-                  id
-                }
-              }
-            }
-        """).safe_substitute({'routeId': self.route_id})
-
-    def handle_json_response(self, json_response):
-        pass
-
-
-class CarDeleter(Query):
-    def __init__(self, car_id: int, endpoint, login_cookie):
-        super().__init__(endpoint, login_cookie)
-        self.car_id = car_id
-
-    def get_query(self) -> str:
-        templ = Template("""
-            mutation DeleteCar{
-              CarMutation{
-                deleteCar(carId: $carId){
-                  id
-                }
-              }
-            }
-        """).safe_substitute({'carId': self.car_id})
-        return templ
-
-    def handle_json_response(self, json_response):
-        pass
-
-
-class StationDeleter(Query):
-    def __init__(self, station_id: int, endpoint, login_cookie):
-        super().__init__(endpoint, login_cookie)
-        self.station_id = station_id
-
-    def get_query(self) -> str:
-        return Template("""
-            mutation DeleteStation{
-              StationMutation{
-                deleteStation(stationId: $stationId){
-                  id
-                }
-              }
-            }
-        """).safe_substitute({'stationId': self.station_id})
-
-    def handle_json_response(self, json_response):
+    def handle_json_response(self, json_response: dict) -> None:
         pass
 
 
-def delete_all(endpoint, login_cookie):
-    """Doesn't delete users! Orders are deleted automatically after all other things deleted (I hope...)"""
+def delete_all(endpoint: str, login_cookie: Cookie) -> None:
```

Comment:
it would probably be better if `delete_all` was moved to separate file `fleet/query/utils.py`